docs/vocs/docs/pages/book/advanced-usage/sdk.mdx

```diff
@@ -160,3 +183,35 @@ The halo2 aggregation proving key `agg_halo2.pk` used in the above example is la
 :::
 
 Note that `DEFAULT_PARAMS_DIR` is the directory where Halo2 parameters are stored by the `cargo openvm setup --evm` CLI command. For more information on the setup process, see the `EVM Level` section of the [verify](/book/writing-apps/verifying-proofs) doc.
+
+## Another Example: Using the Standard SDK
+
+For many programs with modular arithmetic and/or cryptographic operations, the RISC-V instruction set alone may not be enough to achieve the performance desired. OpenVM supports many extensions to accelerate said operations, a standard set of which are included inside the standard SDK configuration.
+
+Take, for example, this toy program that checks if several guest library implementations of [`IntMod`](/book/acceleration-using-extensions/algebra#available-traits-and-methods) are fields by loosely checking if their moduli are prime.
+
+```rust
+// [!include ~/snippets/examples_sdk_guest/little/src/main.rs]
+```
+
+Its `openvm.toml`, as with `Sdk::riscv32()` above, must follow a [specific set configuration](/book/advanced-usage/sdk#standard-sdk-configuration-and-supported-extensions).
+
+Simply initialize `Sdk` using `Sdk::standard()` and build, prove, and verify as above.
+
+```rust
+// [!include ~/snippets/examples_sdk/sdk_app.rs:init]
+```
+
+### Standard SDK Configuration and Supported Extensions
+
+The `app_vm_config` section of the `openvm.toml` of any guest program that uses `Sdk::standard()` must be the following:
+
+```toml [openvm_standard.toml]
+// [!include ~/snippets/examples_sdk_toml/openvm_standard.toml]
+```
+
+Observe that this standard `openvm.toml` also enables normal Rust and `openvm::io` functions (via the `rv32i`, `rv32m`, and `io` extensions). `keccak` and `sha256` enable intrinsic instructions for the [Keccak](/book/acceleration-using-extensions/keccak) and [SHA-256](/book/acceleration-using-extensions/sha-256) hashes respectively, and `bigint` supports [Big Integer](/book/acceleration-using-extensions/big-integer) operations.
+
+[Modular](/book/acceleration-using-extensions/algebra) operations for the BN254, Secp256k1 (i.e. K256), Secp256r1 (i.e. P256), and BLS12-381 moduli are also supported, as well as [Complex Field Extension](/book/acceleration-using-extensions/algebra#complex-field-extension) operations over the BN254 and BLS12-381 prime fields. [Elliptic Curve Cryptography](/book/acceleration-using-extensions/elliptic-curve-cryptography) operations (using preset points) are also supported for the BN254, Secp256k1, Secp256r1, and BLS12-381 curves, and [Elliptic Curve Pairing](/book/acceleration-using-extensions/elliptic-curve-pairing) checks are supported for the BN254 and BLS12-381 curves.
```

Comment:
for modular: BN254, K256 etc are elliptic curves -- you need to specify that the moduli of the scalar field and coordinate field of each curve are supported

Reply:
complex field extensions are only for the coordinate fields of those curves